src/cosmosdb-preview/linter_exclusions.yml

```diff
@@ -24,6 +24,9 @@ cosmosdb create:
     gremlin_databases_to_restore:
       rule_exclusions:
       - option_length_too_long
+    enable_priority_based_execution:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
instead of adding linter exclusion, can you add a short option for this parameter?

Reply:
The name of the feature is priority based execution. It would be clearer if we used this parameter name. I see there are other parameters like enable_multiple_write_locations, enable_automatic_failover, etc which have added the exclusion as well? Can we go ahead with this name itself for customer clarity?

Reply:
I'm not saying replace the original name, but just adding another option. eg, customer can pass `--enable-priority-based-execution` or `--enable-pbe` or some name else